module1/homework/removeVowels/vowel.cpp

```diff
@@ -0,0 +1,14 @@
+#include "vowel.hpp"
+
+#include <algorithm>
+
+void removeVowels(std::vector<std::string>& words) {
+    const std::string vowels = "aeiouyAEIOUY";
```

Comment:
Have you considered putting this in the header. Since you would to have const like this easier to access.
Maybe you should use constrexpr instead since you know value before compiling.